recipes/reprexpy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "reprexpy" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 847dd3a6584b26c8f02f5a61f23b3d548d745a56da4ad6788c1e3129a8a0d6ca
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    # This currently a hard dependency on installation.
+    # Can be removed once https://github.com/crew102/reprexpy/pull/3 is merged
+    - pytest-runner
+  run:
+    - asttokens
+    - ipykernel
+    - ipython
+    - matplotlib
+    - nbconvert
+    - nbformat
+    - pyimgur
+    - pyperclip
+    - python
+    - setuptools
+    - stdlib-list
+    - tornado <=5.1.1
+
+test:
+  imports:
+    - reprexpy
+  requires:
+    - pickledb
+    - pytest
+    - pyzmq
```

Comment:
If the import requires these things, then they are dependencies at run time. I suspect however this section can be removed.

Reply:
No, these are actual test dependencies. They are used for testing the reproducible example this package generates.